utilities.py

```diff
@@ -10,9 +10,11 @@
 def setup_nltk():
     import nltk
 
-    nltk.download("punkt")
-    nltk.download("averaged_perceptron_tagger")
-    nltk.download("stopwords")
+    punkt = nltk.download("punkt")
+    averaged_perceptron_tagger = nltk.download("averaged_perceptron_tagger")
+    stopwords = nltk.download("stopwords")
+
+    return all((punkt, averaged_perceptron_tagger, stopwords))
```

Comment:
Returning a boolean status of whether all packages were successfully downloaded or not.